### ISSUE TITLE
inception-v3 get 0% accracy on ILSVRC2012_val.txt?

### ISSUE THREAD

**Issue by chunniunai220ml** [soeaver/caffe-model#68]
the  inception-v3 get 0% accracy on ILSVRC2012_val.txt,I have set mean=128.0,std=128.0.
v3 must test in ILSVRC2015_val????? as it trained on  ILSVRC2015 dataset???
and  ILSVRC2012 is diiferent from  ILSVRC2015 on cls setting?

**Comment by soeaver** [comment]
yes, the groundtruth of ILSVRC2015 is different with ILSVRC2012

**Comment by chunniunai220ml** [comment]
@soeaver  thanks for your response timely!
so,you trained inception v3 on ILSVRC2015?
and i can finetune on ILSVRC2012 by your provided v3 pre-trained model,and then test ILSVRC2012_val dataset?

**Comment by soeaver** [comment]
You can finetune the model on ILSVRC2012, but I don't think it's necessary, the images of ILSVRC2012 and ILSVRC2015 are exactly the same, only the order of the labels is different.

**Comment by chunniunai220ml** [comment]
@soeaver  can you share your solver about inceptionV3?and i just add the data_layer and loss layer to finetune inception model?

**Comment by THU-cui** [comment]
@chunniunai220ml 
hi,  do you have solved this problem? i also encountered it. Can i test this caffemodel on ILSVRC2012 without finetune?

**Comment by chunniunai220ml** [comment]
> @chunniunai220ml
> hi, do you have solved this problem? i also encountered it. Can i test this caffemodel on ILSVRC2012 without finetune?

**Comment by chunniunai220ml** [comment]
@THU-cui 
I tested the model on ILSVRC2015 as the two dataset(12 and 15)are very similiar content just classified a bit different.
